addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -3173,6 +3147,39 @@ export class OdooEditor extends EventTarget {
         }
     }
 
+    /**
+     * Inserts a link in the editor. Called after pressing space or (shif +) enter.
+     * Performs a regex check to determine if the url has correct syntax.
+     */
+    _insertLink() {
```

Comment:
I disagree with the naming of this methog because it does not always insert a link. I suggest something more along the lines of `_handleAutomaticLinkInsertion`.

Reply:
Done.